helm-charts/templates/deployment.yaml

```diff
@@ -50,6 +50,10 @@ spec:
         {{- include "t2ps-chatbot-ui.combinedImagePullSecrets" . | nindent 8 }}
       {{- end }}
       volumes:
+        - name: run
+          emptyDir: {}
+        - name: nginx-temp
+          emptyDir: {}
```

Comment:
Maybe add something in values.yaml in case we need to increase the default size.